W-16159253-rpamodel-gatewaydefaultpath-se (#397)
mulesoft/docs-rpa: rpa-builder/6.6/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc

image:toolbox-activity-parameters.png[The Activity Parameters Action Step, 40%, 40%]

> Only one _Activity Parameters_ Action Step can exist in a process and it's automatically placed in the _Workflow Initialization_ section when creating a new Workflow.

Reviewer: ```suggestion
A process can have one _Activity Parameters_ Action Step and it's automatically placed in the _Workflow Initialization_ section when creating a new Workflow.
```

Author: Replaced 'process' with 'workflow', because a process can contain several workflows and each of them can contain one AP AS

image:business-process-initialization-window.png[The business process initialization window, 75%, 75%]

> You can only delete an Activity Parameter if it's not being used in a Workflow. If the Activity Parameter is already connected to at least one Workflow, a pop-up message appears indicating that the parameter is still in use.

Reviewer: ```suggestion
You can delete an Activity Parameter if it's not being used in a Workflow. If the Activity Parameter is connected to at least one Workflow, a pop-up message indicates that the parameter is still in use.

```

image:set-expression-condition-for-gateway.png[The Set Condition Expression for Gateway window, 60%, 60%]
. Click *Add Expression* and build an expression by using the available Activity Parameters.

> After you have added all necessary condition expressions, ensure that the last Sequence Flow has an empty condition expression. This is the default flow when all other conditions aren't fullfilled. If there is no default sequence flow, add one to the model.

Reviewer 2: ```suggestion
After you add all necessary condition expressions, ensure that the last sequence flow has an empty condition expression. This is the default flow when all other conditions aren't fulfilled. If there is no default sequence flow, add one to the model.
```
